src/libraries/System.Console/src/System/ConsoleKey.cs

```diff
@@ -5,6 +5,7 @@ namespace System
 {
     public enum ConsoleKey
     {
+        None = 0x0,
```

Comment:
@carlossanlop, do these need XML docs?

Reply:
Area owners can document these new enum values in dotnet-api-docs directly.

Assuming this change makes it into Preview3, that build will get released on April 11: https://dev.azure.com/devdiv/DevDiv/_wiki/wikis/DevDiv.wiki/31692/NET-8-Schedule . On that day, we will update dotnet-api-docs to start showing Preview3 in the MS Docs .NET version dropdown. When that happens, area owners can document the new enums in the dotnet-api-docs xmls:

- https://github.com/dotnet/dotnet-api-docs/blob/6a16408631322b76de318358be710f8dfbbbb5c1/xml/System/ConsoleKey.xml
- https://github.com/dotnet/dotnet-api-docs/blob/6a16408631322b76de318358be710f8dfbbbb5c1/xml/System/ConsoleModifiers.xml

By the way, I don't know if you also wanted to know this, but fortunately all the old enum values area already documented in dotnet-api-docs:

- https://learn.microsoft.com/en-us/dotnet/api/system.consolekey?view=net-7.0
- https://learn.microsoft.com/en-us/dotnet/api/system.consolemodifiers?view=net-7.0

So only the new two enum values will need docs.

Reply:
So the process is we merge this first, tooling will update the XML files and then I can submit a PR to add the text?

Reply:
The typical process is we use xml comments on new apis and then those are used to seed the docs. It sounds like Carlos is saying you could also wait and submit a pr to the docs repo for these later.

Reply:
Sounds good. I also submitted a PR for that; if that's the wrong way of doing it, I can also wait until the tooling populated the files for the new members.